shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeD.test.ts

```diff
@@ -2,14 +2,28 @@ import { CourtIssuedDocumentFactory } from './CourtIssuedDocumentFactory';
 import {
   calculateISODate,
   createISODateString,
+  FORMATS,
+  getBusinessDateInFuture,
 } from '../../utilities/DateHandler';
 import { getTextByCount } from '../../utilities/getTextByCount';
 
+const oneMonthFromNow = getBusinessDateInFuture({
+  numberOfDays: 30,
+  outputFormat: FORMATS.ISO,
+  startDate: createISODateString(),
+});
+
+const dateInDashedFormat = getBusinessDateInFuture({
```

Comment:
I think this variable can be a smidge more descriptive:

```suggestion
const oneMonthFromNowHumanized = getBusinessDateInFuture({
```